ongebo/java-coders-nairobi
# #155738437 Initialise repository as an Android Project

### What does this PR do?
Sets up the basic file structure for an Android Project
#### Description of the task to be completed?
- Add README.md, .gitignore, and LICENSE.txt files
- Autogenerate file structure using Android Studio
#### How should this be manually tested?
- Clone the repository and check out this branch
- Open the project in Android Studio and run the app
- Verify that a screen with the text `Hello World!` and title `Java Coders Nairobi` is displayed
#### Screenshot
![image](https://user-images.githubusercontent.com/34346070/53231713-ab669100-369a-11e9-8043-0512cb1deba4.png)
#### What are the relevant Pivotal Tracker stories?
[#155738437](https://www.pivotaltracker.com/story/show/155738437)
